src/test/java/core/basesyntax/service/FruitServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.strategy.FruitHandler;
+import core.basesyntax.strategy.FruitHandlerStrategy;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
```

Comment:
As far as I remember it was said that you should use Junit5 for testing, and I don't know if Mockito is allowed in this hw, so wait for one of the mentors to tell you